src/olympia/abuse/models.py

```diff
@@ -261,6 +267,11 @@ def process_decision(
     def appeal(self, *, abuse_report, appeal_text, user, is_reporter):
         if not self.can_be_appealed(is_reporter=is_reporter, abuse_report=abuse_report):
             raise CantBeAppealed
+        # Author appeals can be done without an abuse report. Unfortunately
```

Comment:
That only moves the problem to the task (which I'd like to avoid - the tasks are fairly lightweight, not much code, with the logic in the `CinderJob` methods, and it's probably best to keep them that way)